### PR TITLE
render signal_box at zoom 11/12 as point w/o text

### PR DESCRIPTION
**Pull request by bjoernma**
This PR resolves issue #9 . I dont think that it's necessary to have text as it would be crowded on the map.

### CODE REVIEW

File: signal_boxes.mss [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
+#signal_boxes_point[zoom>=11] {
   /* render a circle, therefore marker-file is not set */
   marker-fill: @signal_box_color;
   marker-line-width: 1;
   marker-line-color: @signal_box_casing_color;
   marker-width: @signal_box_size;
   marker-height: @signal_box_size;
   [zoom>=15] {
-    marker-width: @signal_box_size_z15;
+    marker-width: @signal_box_size_z16;
```
Comment (Nakaner) [inline signal_boxes.mss:24]: Carto error:

`Error: signal_boxes.mss:24:18 variable @signal_box_size_z16 is undefined`

**Review by Nakaner — CHANGES_REQUESTED** [review]

**Comment by DerDakon** [comment]
@bjoernma what about the carto error reported above?

**Comment by bjoernma** [comment]
I cant see why I wanted to change the size at z16 as well, but so the error should be gone.

**Review by DerDakon — APPROVED** [review]

**Comment by Nakaner** [comment]
Rendering results after my patch:

Zoom level 11, Maschen Rbf:

maschen-z11

Zoom level 12, Maschen Rbf:

maschen-z12

**Review by Nakaner — APPROVED** [review]

**Comment by DerDakon** [comment]
This has been merged, no idea why the web interface doesn't get it.

**Pull request closed without merging**